lib/util/hud.rb

```diff
@@ -2159,6 +2159,10 @@ def coc_name(coc_code)
     cocs.try(:[], coc_code) || coc_code
   end
 
+  def valid_coc?(coc_code)
+    cocs.key?(coc_code)
+  end
+
   def cocs
```

Comment:
How often does HUD change this list? Should it be moved to the DB?

Reply:
I don't believe they change very often, and yes, eventually they should probably end up in the DB, for a variety of reasons, but for now, I think in the file is ok.